parm/archive/gdasocean_analysis.yaml.j2

```diff
@@ -0,0 +1,24 @@
+gdasocean_analysis:
+    {% set head = "gdas.t" + cycle_HH + "z." %}
+    name: "GDASOCEAN_ANALYSIS"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gdasocean_analysis.tar"
+    required:
```

Comment:
I see three missing files when I compare 
/scratch1/NCEPDEV/global/CI/2588/RUNTESTS/COMROOT/C48mx500_3DVarAOWCDA_ef56339a/logs/2021032418/gdasarch.log
with 
/scratch2/NCEPDEV/stmp1/David.Huber/archive_rotdir/arch_c48mx500_3dvaraowcda/2021032418/gdasocean_analysis.tar.listing
They are: 

HTAR: a   gdas.20210324/18/analysis/ocean/gdas.t18z.ocn.adt_rads_all.stats.csv
HTAR: a   gdas.20210324/18/analysis/ocean/gdas.t18z.ocn.icec_amsr2_north.stats.csv
HTAR: a   gdas.20210324/18/analysis/ocean/gdas.t18z.ocn.icec_amsr2_south.stats.csv

@AndrewEichmann-NOAA @guillaumevernieres can you confirm that these should be archived.

Reply:
Yes, they should.

Reply:
Alright, thanks, I'll add them in.